src/modules/universal-discovery/components/content-meta-preview/content.meta.preview.component.tsx

```diff
@@ -1,9 +1,29 @@
-import React, { Component, Fragment } from 'react';
-import PropTypes from 'prop-types';
+import * as React from 'react';
+import { Component, Fragment } from 'react';
 
 import './css/content.meta.preview.component.css';
 
-export default class ContentMetaPreviewComponent extends Component {
+type ContentMetaPreviewComponentProps = {
```

Comment:
There are 3 possibilities when it comes to converting propTypes:
1) We can convert `MyComponent.propTypes` to type passed to `Component<Props, State>`. This gives compile time safety + enables prompting properties by the editor during writing code.
2) We can leave old prop types and just pass `any` to `Component`: `Component<any, any>` - no checking by TS, but still we have dynamic checking during use of the application. (as in e.g. `content.creator.component.tsx` in this PR)
3) Both above options - leave propTypes and add TS types